### PR TITLE
Fixes #10688: Use "-" in place of "_" in named reference in doc

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/10688

### CODE REVIEW

File: 10_installation/05_requirements/00_intro.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+[[rudder-installation-requirements]]
+=== Requirements
```
Comment (amousset) [inline 10_installation/05_requirements/00_intro.txt]: newline :)

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
PR rebased

**Pull request merged**